tree/tree.h

```diff
@@ -2,6 +2,7 @@
 #define  _MACHINELEARNING_TREE_TREE_H__
 
 #include "splitter.h"
+//#include "criterion.h"
 
 #include <vector>
 
```

Comment:
This isn't a base class anymore. Could you please change the comment to reflect that?